Makefile

```diff
@@ -110,3 +120,17 @@ else
 endif
 endif
 endif
+
+ci-setup:
+	@echo "Installing Go..."
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ${GOPATH}/bin v1.20.0
```

Comment:
We have variables for the `gox` and `golangci-lint` versions that we can re-use here :+1: